src/main/java/com/talentpath/airTNB/models/Description.java

```diff
@@ -0,0 +1,69 @@
+package com.talentpath.airTNB.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import javax.persistence.*;
+@Entity
+@Table(name="descriptions")
+public class Description {
+        @Id
+        @GeneratedValue
+        private Integer id;
+        
+        @OneToOne(cascade = CascadeType.ALL)
+        @JsonIgnore
+        private Listing listing;
+        
+        private String smallDescription;
+        private String cancellationDate;
+
+        // Is this in the listing?
```

Comment:
I think it is in Listing under as "subtitle" but I am not 100% sure. Think it would make sense to have it here though

Reply:
These aren't my changes, I merged in someone else's changes from staging :)